kimenyikevin/FreeMentor
# #168237345 add readme file on repository and also edit link in swagger documentation

#### What does this PR do?
- have readme on the root folder of repository and working APIs documentation
#### Description of Task to be completed?
 have the following works properly 
- The ReadMe on the root of the repository
- documentation be able to fetch data on Heroku
#### How should this be manually tested?
- visit FreeMentor repository to see ReadMe 
- And also click on this link to view documentation https://freementorss.herokuapp.com/api-docs/
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#168237345
#### Screenshots (if appropriate)
#### Questions:
